apps/ebclfsa-demo/hi/hi_main.c

```diff
@@ -30,7 +30,7 @@ static void check_and_signal_watchdog(struct shmlib_watchdog* const watchdog,
     }
 }
 
-static int start_hi_forward_child(void*)
+static int start_hi_forward_child(void *arg __attribute__((unused)))
```

Comment:
Let's use modern C++ here. The maybe_unused attribute should be supported by clang

```suggestion
static int start_hi_forward_child(void *arg [[maybe_unused]])
```

Reply:
In the current Clang version 14, [[maybe_unused]] is not implemented for C. It is supported starting from Clang 18 and later versions that support -std=c23.

Reply:
I was tasting with a C++ file, where it is supported almost since clang existed...